### DISCUSSION TITLE
processing TIFs with alpha channels, made from PDF w Spot Colors

### DISCUSSION THREAD

**Discussion by fuzzy-andreas** [ImageMagick/ImageMagick#7428] [Help] [answered]
### ImageMagick version

7.1.1-Q16-HDRI

### Operating system, version and so on

Windows Server 2016 Std/CLI

### Description

Hello, I'm new here and have a question about processing TIFs with alpha channels!
I use IM under Enfocus Switch for Windows with
-distort arc "43 359 12000 8500"
to distort images.
Now I've received a PDF with spot colors and it doesn't distort my from the PDF generated TIF with alpha channels (examples attached).
Can anyone tell me how I can get a TIF or PNG with spot colors from the PDF that IM distorts correctly?
I have Adobe CC and other tools at my disposal.

It would be great if I could get some help here!
[spot_color.zip](https://github.com/user-attachments/files/16014582/spot_color.zip)

**Comment by snibgo** [comment, marked as answer]
IM doesn't read spot colours.

**Reply by fuzzy-andreas** [reply]
Hello, Thanks for the quick reply. That saves me a long search!
